api/pkg/database/main.go

```diff
@@ -34,12 +33,19 @@ var (
 	}
 )
 
+func init() {
+	DB = initDataBase(DBPath)
+	go StartEnforcingDatabaseSize()
+}
+
 func GetEntriesTable() *gorm.DB {
 	return DB.Table("mizu_entries")
 }
 
 func initDataBase(databasePath string) *gorm.DB {
-	temp, _ := gorm.Open(sqlite.Open(databasePath), &gorm.Config{})
+	temp, _ := gorm.Open(sqlite.Open(databasePath), &gorm.Config{
+		Logger: logger.Default.LogMode(logger.Silent),
```

Comment:
gorm's logger prints full entry objects on expected errors (locked db when pruning), this makes mizu stuck

Reply:
What other way do we have to know about the errors?

Reply:
Implemented a truncating logger